HealthcareSystem/PatientWebApp/Validators/PatientValidator.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Backend.Model.Exceptions;
+using Backend.Service;
+using Backend.Service.UsersAndWorkingTime;
+using PatientWebApp.DTOs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace PatientWebApp.Validators
+{
+    public class PatientValidator
+    {
+        public PatientValidator() { }
+
+        public void validatePatientFields(PatientDTO patientDTO)
```

Comment:
Maybe should be good to make these small if(){..} to little private methods? That would be descriptive and cleaner.

Reply:
Ok, I'll create private methods when I'm sure which checks I still need. For now, I will merge this branch with develop.